pages/windows/dir.md

```diff
@@ -0,0 +1,19 @@
+# DIR 
+
+> Displays file and folders in current directory or given directory.
```

Comment:
How about "List directory contents" ? This matches with the description of `ls`

Reply:
yeah, sure